contrib/sway-session.target

```diff
@@ -0,0 +1,5 @@
+[Unit]
+Description=sway session
```

Comment:
While we're waiting for this to get merged, do you mind adding an extra line here?
```suggestion
Description=sway session
Documentation=man:systemd.special(7)
```